packages/SwingSet/src/kernel/kernel.js

```diff
@@ -395,6 +395,7 @@ export default function buildKernel(kernelEndowments) {
         throw Error(`unable to process message.type ${message.type}`);
       }
       commitCrank();
+      kernelKeeper.incrementCrankNumber();
```

Comment:
Don't we need to include the newly-incremented crank number in the DB commitment? So maybe these two need to be done in the opposite order?

Reply:
Your intuition would suggest that and your intuition would be wrong.  (My intuition suggested that too until I debugged it.  Bad intuition!  Bad!  Bad!).  See my comment on your next comment.